packages/jest-jasmine2/src/extend_jasmine_expect.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
i mean... the company can't be 100% perfect! i knew there was a flaw somewhere! :)